projects/1.20-fabric/assets/furnection/ls_furniture/lang/zh_cn.json

```
@@ -0,0 +1,750 @@
+  "container.ls_furniture.dark_oak_chestDouble": "大型深色橡木箱子",
+  "container.ls_furniture.mangrove_chest": "红树木箱子",
+  "container.ls_furniture.mangrove_chestDouble": "大型红树林木箱子",
+  "container.ls_furniture.cherry_chest": "樱花木箱子",
+  "container.ls_furniture.cherry_chestDouble": "大型樱花木箱子",
+  "container.ls_furniture.bamboo_chest": "竹箱子",
+  "container.ls_furniture.bamboo_chestDouble": "大型竹箱子",
+  "container.ls_furniture.crimson_chest": "绯红木箱子",
+  "container.ls_furniture.crimson_chestDouble": "大型绯红木箱子",
+  "container.ls_furniture.warped_chest": "绯红木箱子",
+  "container.ls_furniture.warped_chestDouble": "大型诡异木箱子",
+
+
+  "subtitles.block.workstation.ui.take_result": "工作站：合成",
+  "subtitles.item.tool.use": "工具：已使用",
+  "subtitles.block.blinds.use": "百叶窗：切换",
+  "subtitles.block.drawer.close": "抽屉：关闭",
+  "subtitles.block.drawer.open": "抽屉：开启",
+  "subtitles.block.chair-stool.untuck": "椅子：解开",
+  "subtitles.block.chair-stool.tuck": "椅子：收起",
+  "subtitle.ls_furniture.attach_to_letter": "邮件：已添加物品",
+  "subtitle.ls_furniture.detach_from_letter": "邮件：已移除附件",
+
+
+  "tooltip.ls_furniture.screen.shift": "§eShift§r了解更多信息",
+  "tooltip.ls_furniture.screen.ctrl": "§eCtrl§r了解更多信息",
+  "tooltip.ls_furniture.screen.alt": "§eAlt§r了解更多信息",
+  "tooltip.ls_furniture.screen.shift.ctrl": "§Shift-Ctrl§r了解更多信息",
+  "tooltip.ls_furniture.screen.blank": "",
+  "tooltip.ls_furniture.screen.properties": "§7属性:",
+  "tooltip.ls_furniture.screen.divider": "§9--------------",
+  "tooltip.ls_furniture.screen.divider2": "§7__| 属性 |__",
+
+  "tooltip.ls_furniture.saw": "§e右击§r更改材质",
+  "tooltip.ls_furniture.hammer": "§e右击§r更改家具模型",
+  "tooltip.ls_furniture.hammer.shift": "§eShift-右击§r循环浏览家具可变属性",
+  "tooltip.ls_furniture.wrench": "§e右击§r旋转方块",
+  "tooltip.ls_furniture.wrench.shift": "§eCRTL-右击§r锁定家具状态",
+  "tooltip.ls_furniture.shears": "§e右击§r移除坐垫 / 羊毛",
+  "tooltip.ls_furniture.sink": "对柜台§e右击§r",
+  "tooltip.ls_furniture.oven": "对柜台§e右击§r",
+  "tooltip.ls_furniture.oven_vent": "对吧台§e右击§r",
+  "tooltip.ls_furniture.fire_place": "用打火石§e右击§r点燃",
+  "tooltip.ls_furniture.fire_place_off": "空手§e右击§r熄灭",
+  "tooltip.ls_furniture.tombstone": "§eShift-右击§r隐藏泥土",
+
+
+  "tooltip.ls_furniture.connectable": "§2可连接",
+  "tooltip.ls_furniture.lockable": "§2可锁定",
+  "tooltip.ls_furniture.sawable": "§2可锯碎",
+  "tooltip.ls_furniture.hammerable": "§2可锤击",
+  "tooltip.ls_furniture.wrenchable": "§2可扳动",
+  "tooltip.ls_furniture.dyeable": "§2可染色",
+  "tooltip.ls_furniture.shulkerbox": "§2类潜影盒",
+
+  "tooltip.ls_furniture.letter_add": "§eShift-右击§r将手持物品附在信件上",
+  "tooltip.ls_furniture.letter_remove": "§eShift-右击§r从信件中移除所附物品",
+  "msg.ls_furniture.letter.attachment_added": "将%s附加到信件中",
+  "msg.ls_furniture.letter.attachment_removed": "已移除附件",
+  "tooltip.ls_furniture.letter.author": "由%s发送",
+  "tooltip.ls_furniture.letter.attachment": "附件：%s",
+
+
+  "msg.ls_furniture.mailbox.ownership_info": "再次右击以获取所有权",
+  "msg.ls_furniture.mailbox.new_mail_loc": "在 % 处收到邮件",

```

这个键在游戏内显示效果如何
```suggestion
  "msg.ls_furniture.mailbox.new_mail_loc": "在%s处收到邮件",
```
改为这样呢